lib/active_record_doctor/detectors/missing_foreign_keys.rb

```diff
@@ -49,6 +50,18 @@ def polymorphic_foreign_key?(table, column)
           another_column.name == type_column_name
         end
       end
+
+      def model_destroyed_async?(table, column)
+        # Check if there are any models having `has_many ..., dependent: :destroy_async`
+        # referencing the specified model.
```

Comment:
"referencing the specified table and column" - this method doesn't accept `model` as an argument